lib/model/content.dart

```diff
@@ -986,18 +986,18 @@ class _ZulipContentParser {
           && divElement.className == "codehilite");
 
       if (divElement.nodes.length != 1) return null;
-      final child = divElement.nodes[0];
+      final child = divElement.nodes.first;
```

Comment:
If we use a fancy getter here, it should be `single` — that way we correctly express the expectation that this is the only node (not the first of potentially several nodes).